src/app/components/forms/login-form/login-form.component.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../../styles/colors';
+
+
+.login-form-button-container {
+  text-align: center;
+  margin-top: 20px;
+
+  .error-message {
+    color: $color-cerise-500;
+  }
+
+  .login-form-button {
+    margin: 10px auto;
```

Comment:
Properties should be ordered min-width, margin

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)